wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests+Participants.swift

```diff
@@ -555,7 +555,7 @@ final class ConversationParticipantsTests: ZMConversationTestsBase {
 
     func testThatItRefetchesRolesIfNoRoles() {
 
-        syncMOC.performGroupedAndWait { _ -> Void in
```

Comment:
Nice, however, maybe these changes should have gone into a separate PR.

Reply:
The rule got rewritten with version 0.54.0 and thus lead to failure. Either I could change the swiftlint configuration or just solve the cases...